hummingbot/client/config/config_helpers.py

```diff
@@ -54,9 +56,9 @@ def parse_cvar_value(cvar: ConfigVar, value: any):
             logging.getLogger().error(f"\"{value}\" is not an integer.")
             return 0
     elif cvar.type == 'bool':
-        if type(value) == str and value.lower() in ["true", "yes"]:
+        if type(value) == str and value.lower() in ["true", "yes", "y"]:
```

Comment:
i think isinstance(value, str) is stand for type checking

Reply:
resolve